Library/Homebrew/extend/git_repository.rb

```diff
@@ -13,6 +13,13 @@ def git_origin
     end
   end
 
+  def git_origin=(origin)
+    return unless git? && Utils.git_available?
```

Comment:
Make if rather than unless

Reply:
@reitermarkus Could update them too; it's pretty yuck.

Reply:
On another note, would we even get to a point where we call any of these when `Utils.git_available?` is `false`? I mean, how can there be a `.git` directory when `git` is not available?